backend/src/test/java/com/example/subjecthub/SubjectServiceControllerTests.java

```diff
@@ -138,6 +146,134 @@ public void testGetSubjectsFuzzySearch() throws Exception {
     }
 
```

Comment:
Could you please add a testGetComments test that ensures we can successfully retrieve a list of comments?